blurring_as_a_service/settings/attr_dict.py

```diff
@@ -32,6 +32,8 @@ def _data_from_dict(cls, d: Dict):
         for k, v in d.items():
             if isinstance(v, dict):
                 data[k] = cls(v)
+            elif isinstance(v, str):
+                data[k] = v  # Skip the formatting step for strings # TODO Sebastian
```

Comment:
This seems kind of superfluous. I guess it's a tiny optimization, but still, `data[k]` is set to `v` on line 46 if type != list. What does Sebastian still have to do?

Reply:
I had some issues when using a format like this in the yaml file:
  input_container_root: "azureml://subscriptions/{subscription}/resourcegroups/{resourcegroup}/workspaces/{workspace}/datastores/{datastore_name}/paths/"


It would also try to parse the {subscription} {resourcegroup} etc. And throw an error. I now ignore this parsing when the value is a string. But, Im not sure if this is the best solution. Better to discuss it with Sebastian next week I think

Reply:
@thomasbrockmeier-ams  Discussed with Sebastian, we dont use the parsing of strings in the YAML file. So, I commented a part of the code in settings_helper.py